tests/data-files/S2A_OPER_MSI_L2A_TL_VGS1_20220401T110010_A035382_T34LBP/expected_output.json

```diff
@@ -1803,6 +1803,72 @@
         "reflectance"
       ]
     },
+    "cloud": {
```

Comment:
why were these added to only one of the many test files?

Reply:
Hmm, good question, I used https://github.com/stactools-packages/sentinel2/blob/main/scripts/create_expected.py to update the test files which does exclude some.  And then some of those tests are for the SAFE archives, and this only adds assets when using the s3 bucket.